### PR TITLE
DOC: Fixing EX01 - Added examples

### PR DESCRIPTION
**Pull request by DeaMariaLeon**
- [x] All [code checks passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit).

Towards https://github.com/pandas-dev/pandas/issues/37875

### CODE REVIEW

**Review by MarcoGorelli — CHANGES_REQUESTED** [review]
Nice examples! Just got a small comment, then this can go in

File: pandas/_libs/tslibs/period.pyx [hunk 1]
```diff
@@ -1661,6 +1661,21 @@ cdef class PeriodMixin:
...
+        >>> s
+        0   2020-01
+        1   2020-02
+        2   2020-03
+        dtype: period[M]
+        >>> s.dt.end_time
+        0   2020-01-31 23:59:59.999999999
+        1   2020-02-29 23:59:59.999999999
+        2   2020-03-31 23:59:59.999999999
+        dtype: datetime64[ns]
```
[the remark this answers is not in the document]
Comment (DeaMariaLeon) [reply pandas/_libs/tslibs/period.pyx]: Changed, thanks. 
I meant to add an example for `Series.dt.end_time` so, I put it on the wrong file!

File: pandas/core/arrays/datetimes.py [hunk 2]
```diff
@@ -564,6 +564,17 @@ def tz(self) -> tzinfo | None:
         -------
         datetime.tzinfo, pytz.tzinfo.BaseTZInfo, dateutil.tz.tz.tzfile, or None
             Returns None when the array is tz-naive.
+
+         Examples
+        --------
```
Comment (MarcoGorelli) [inline pandas/core/arrays/datetimes.py:569]:
```suggestion
        Examples
        --------
```

**Review by MarcoGorelli — APPROVED** [review]
thanks @DeaMariaLeon !

**Comment by DeaMariaLeon** [comment]
Thanks!

**Pull request merged**